docs/tutorial/tutorial-3-preload.md

```diff
@@ -38,7 +38,25 @@ called a **preload**.
 ## Augmenting the renderer with a preload script
 
 A BrowserWindow's preload script runs in a context that has access to both the HTML DOM
-and a Node.js environment. Preload scripts are injected before a web page loads in the renderer,
+and a limited subset of Node.js and Electron APIs.
+
+:::info Preload script sandboxing
+
+From Electron 20 onwards, preload scripts are **sandboxed** by default and no longer have access
```

Comment:
It's not quite accurate to say here that the _preload script_ is sandboxed—the whole process is!
```suggestion
From Electron 20 onwards, renderers are **sandboxed** by default, so preload scripts no longer have access
```